src/app/clients/create-client/create-client.component.ts

```diff
@@ -90,7 +90,8 @@ export class CreateClientComponent {
       locale
     };
     this.clientsService.createClient(clientData).subscribe((response: any) => {
-      this.router.navigate(['../', response.resourceId], { relativeTo: this.route });
+      const url = `/clients/${response.clientId}/loans-accounts/create`
```

Comment:
After a client got created, i dont see the reason why to navigate the customer automatically to the page to create a new loan for that client. Would you please share more details about this?